### PR TITLE
Edits for grammar, capitalization, content, etc.

### PR DESCRIPTION
**Pull request by aarongustafson**

### CODE REVIEW

File: microsoft-edge/progressive-web-apps-chromium/index.md [hunk 1]
```diff
@@ -59,57 +59,55 @@ With [Progressive Web Apps][MDNApps] (or simply *PWA*s) you do not have to decid
     :::column-end:::
 :::row-end:::
 
-By building or converting your existing site to a PWA, you can better engage your existing audience with push notifications, app like integration and offline support.  At the same time, you can continue building your audience on the open web, as users discover your PWA through search and link-sharing.  Best of all, you can update your app by simply updating your web server code.  
+By building or converting your existing site to a PWA, to better engage your existing audience with push notifications, app-like integration and offline support.  At the same time, continue to build your audience on the open web, as users discover your PWA through search and link-sharing.  Best of all, you may update your app by simply updating your web server code.
```
Comment (aarongustafson) [inline microsoft-edge/progressive-web-apps-chromium/index.md:62]: The first sentence has grammar issues

File: microsoft-edge/progressive-web-apps-chromium/index.md [hunk 2]
```diff
@@ -59,57 +59,55 @@ With [Progressive Web Apps][MDNApps] (or simply *PWA*s) you do not have to decid
...
 *   Shared cookie store with the browser profile that installed the app
-*   Access to other browser features via the "..." menu including certificate validation, site permissions, tracking protection and browser extensions
-*   Full access to Microsoft [Edge DevTools][DevtoolsProgressiveWebApps] for debugging your app  
+*   Access to other browser features via the `...` menu including certificate validation, site permissions, tracking protection, and browser extensions
+*   Full access to [Microsoft Edge DevTools][DevtoolsProgressiveWebApps] for debugging your app  
 
 > [!IMPORTANT]
-> To tailor PWAs specifically for Windows 10 that call WinRT APIs from JavaScript, see the documentation specific to the EdgeHTML PWA features here:  [Progressive Web Apps (EdgeHTML)][PwaEdgehtmlIndex].  You can learn about testing your PWA on Windows 10, and distributing it in the Microsoft Store.  
+> To tailor PWAs specifically for Windows 10 that send WinRT API requests from JavaScript, see the [documentation specific to the EdgeHTML PWA features][PwaEdgehtmlIndex].  Learn  more about testing your PWA on Windows 10 and distributing it in the Microsoft Store.
```
Comment (aarongustafson) [inline microsoft-edge/progressive-web-apps-chromium/index.md:82]: "send WinRT API requests" > "make WinRT API requests"

File: microsoft-edge/progressive-web-apps-chromium/index.md (hunk 1 above)
Comment (jm-trd-ms) [reply microsoft-edge/progressive-web-apps-chromium/index.md:62]: Updated.
"By building or converting your existing site to a PWA, you engage better with your existing audience using push notifications, app-like integration, and offline support."

File: microsoft-edge/progressive-web-apps-chromium/index.md (hunk 2 above)
Comment (jm-trd-ms) [reply microsoft-edge/progressive-web-apps-chromium/index.md:82]: Updated. 
"To tailor PWAs specifically for Windows 10 that make WinRT API requests using JavaScript... "

**Comment by aarongustafson** [comment]
LGTM

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**